tests/Makefile.tests

```diff
@@ -32,8 +32,7 @@ LIB_TESTS_L = get_mem_rsv \
 	fs_tree1
 LIB_TESTS = $(LIB_TESTS_L:%=$(TESTS_PREFIX)%)
 
-LIBTREE_TESTS_L = truncated_property truncated_string truncated_memrsv \
-	two_roots named_root
+LIBTREE_TESTS_L = truncated_property truncated_string truncated_memrsv
```

Comment:
Contrary to the commit message, these files are not currently listed in `TESTS_TREES` although they should be, along with all the examples created by `dumptrees`.  Could you updated your patch to add all the missing things to `TEST_TREES` as well.

Reply:
I apologize for any confusion caused by my commit message. To clarify, my intention was to convey that the executables for the two_roots and named_root test cases do not exist. This is distinct from the two_roots.dtb and named_root.dtb files, which are indeed generated and used by other tests. My update was aimed at removing references to the non-existent executables, not the .dtb files you've mentioned. Therefore, our statements do not conflict but address different parts of the testing setup.